docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -120,48 +160,47 @@ Variables are part of a process instance and represent the data of the instance.
 
 ## Step 2: Design a form
 
-You have now designed the process. To allow the user to make the decision, you will now design a [form](../components/modeler/forms/camunda-forms-reference.md). Forms can be added to user tasks and start events to capture user input, and the user input can be used to route the process flow, to make calls to APIs, or to orchestrate your services.
+You have now designed the process. To allow the user to make the decision, you must next design a [form](../components/modeler/forms/camunda-forms-reference.md). Forms are added to user tasks and start events to capture user input, with the user input used to route the process flow, make API requests, or orchestrate your services.
 
 <Tabs groupId="install" className="tabs-hidden">
 <TabItem value="saas">
 
 1. Select the user task you created in **[Step 1](#step-1-create-a-new-process)**.
-2. Click the blue **link icon** in the lower right corner. A menu expands that allows you to create a new form.
-   <img src={ModelerFormMenuImg} style={{width: 400}} alt="Annotation to open the form menu" />
-3. Click **Create new form**. A form will be created and opened in the form editor. The form is automatically named.
+1. Click the blue **link icon** in the lower right corner. A menu expands that allows you to create a new form.
```

Comment:
<img width="432" alt="image" src="https://github.com/user-attachments/assets/dae13207-660c-471f-b82f-2f5d40ad61cc">
I saw this instead of the empty state